tests/test.tracking.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+var tracking = require('tracking');
+
+
+describe('tracking uninitialized', function() {
+
+  beforeEach(function() {
+    this.oldTrackingEnabled = tracking.enabled;
+    this.oldTrackingInitialized = tracking.initialized;
+    tracking.enabled = true;
+    tracking.initialized = false;
+  });
+
+  afterEach(function() {
+    tracking.enabled = this.oldTrackingEnabled;
+    tracking.initialized = this.oldTrackingInitialized;
+  });
+
+  it('should throw when calling setPage', function() {
```

Comment:
Even though this is the first function, it should have ``tracking.initialized = false`` at the top in case someone re-orders the tests later